docs/releases/4.2.txt

```diff
@@ -200,7 +200,17 @@ Signals
 Templates
 ~~~~~~~~~
 
-* ...
+* The ``length_is`` template filter is deprecated in favor of :tfilter:`length`
+  and the ``==`` operator within an :ttag:`{% if %}<if>` tag:
+
+    .. code-block:: html+django
+
+        {# Before: #}
+        {% if value|length_is:"4" %}...{% endif %}
+        {{ value|length_is:"4" }}
+        {# After: #}
+        {% if value|length == 4 %}...{% endif %}
+        {% if value|length == 4 %}True{% else %}False{% endif %}
```

Comment:
This should be in the ["Features deprecated in 4.2"](https://docs.djangoproject.com/en/dev/releases/4.2/#features-deprecated-in-4-2) section.

Reply:
🤦 Of course. I got attracted to "Templates" like a moth to a flame.